src/Raven.Server/Documents/Handlers/IndexHandler.cs

```diff
@@ -378,7 +378,9 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context.Documents, Re
 
                                     var state = x.State;
 
-                                    if (e.IsOutOfMemory() == false && e.IsRavenDiskFullException() == false)
+                                    if (e.IsOutOfMemory() == false && 
+                                        e.IsRavenDiskFullException() == false && 
+                                        e is OperationCanceledException == false) // we probably closed the indexing thread
```

Comment:
Probably better to handle that ex explicitly and not raise any alerts, etc

Reply:
done